### PR TITLE
Install unzip to unzip AWS CLI bundle

### PR DESCRIPTION
**Pull request by lukeseawalker**
Signed-off-by: Luca Carrogu <carrogu@amazon.com>

**Please See** [Git Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions)

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml [hunk 1]
```diff
@@ -64,6 +64,17 @@ phases:
               # Install AWS CLI
               which aws
               if [[ $? -ne 0 ]]; then
+                echo "Installing unzip"
+                if [[ ${PLATFORM} == RHEL ]]; then
```
Comment (yuleiwan) [inline cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml:68]: PLATFORM isn't defined in component.

File: cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml [hunk 2]
```diff
@@ -64,6 +64,17 @@ phases:
...
+                  if [[ ${OS} == centos8 ]]; then
```
Comment (yuleiwan) [inline cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml:69]: we should define OS variable in this step.

File: cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml (hunk 1 above)
Comment (lukeseawalker) [reply cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml]: fixed

File: cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml (hunk 2 above)
Comment (lukeseawalker) [reply cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml]: fixed

**Review by yuleiwan — DISMISSED** [review]

**Review by yuleiwan — APPROVED** [review]

**Pull request merged**